Tests/scripts/hook_validations/image.py

```diff
@@ -86,3 +90,45 @@ def is_existing_image(self):
             return False
 
         return True
+
+    def is_not_default_image(self):
+        """Check if the image is the default one"""
+        if re.match(IMAGE_REGEX, self.file_path, re.IGNORECASE):
+            with open(self.file_path, "rb") as image:
+                image_data = image.read()
+                encoded_image = base64.b64encode(image_data)
+                if isinstance(encoded_image, bytes):
+                    encoded_image = encoded_image.decode("utf-8")
+                if encoded_image == DEFAULT_IMAGE_BASE64:  # disable-secrets-detection
+                    print_error("{} is the default image, please change to the "
+                                "integration image.".format(self.file_path))
+                    self._is_valid = False
+                    return False
+                else:
+                    return True
+
+        else:
```

Comment:
and add comment what this condition handles

Reply:
fixed